CDTDatastore/query/CDTQIndexUpdater.h

```diff
@@ -66,21 +69,21 @@
  The index is assumed to already exist.
  */
 - (BOOL)updateIndex:(NSString *)indexName
-         withFields:(NSArray /* NSString */ *)fieldNames
-              error:(NSError *__autoreleasing *)error;
+         withFields:(NSArray<NSString *> *)fieldNames
+              error:(NSError *__autoreleasing __nullable *__nullable)error;
 
 /**
  Generate the DELETE statement to remove a documents entries from an index.
  */
-+ (CDTQSqlParts *)partsToDeleteIndexEntriesForDocId:(NSString *)docId
-                                          fromIndex:(NSString *)indexName;
++ (nullable CDTQSqlParts *)partsToDeleteIndexEntriesForDocId:(NSString *)docId
```

Comment:
I don't think this can return `nil` now the args are nonnull as they're in an audited section.